grafana/include/airflow-state.json

```diff
@@ -1784,6 +1784,6 @@
   },
   "timezone": "",
   "title": "Airflow State",
-  "uid": "h7dUiVJmz",
+  "uid": "h7dUiVJmq",
```

Comment:
We should keep this the same. If we change it, it can break people's bookmarks. EG: https://grafana.dev.astrodev.io/d/h7dUiVJmz/airflow-state